config/quality/pmd/pmd-ruleset.xml

```diff
@@ -11,7 +11,7 @@
 
     <rule ref="rulesets/internal/all-java.xml/TooManyStaticImports">
         <properties>
-            <property name="legalPackages" type="String"
```

Comment:
This is probably the cause for the build failure. Since `legalPackages` is a custom property it needs to have a defined type.